docs/changelog.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+* **18 Feb 2019:**
+  * Added a `build.sh` script to each worker type so that it is possible to build a single worker instance with the script.
+  * Added a `download_dependencies.sh` script to `SpatialOS/scripts/` for downloading the schema compiler, the standard schema library, and the worker SDK in C# for MacOS, Windows, and Linux.
+  * Added a `generate-schema-descriptor.sh` script to `SpatialOS/scripts/` for generating a schema descriptor from the schemas. 
+  * Added a `utils.sh` script to `SpatialOS/scripts` for storing paths to worker types, tools, and schemas.
+  * Added a `build-project.sh` script to `SpatialOS/scripts/`. It calls other scripts to build the project.
```

Comment:
Also don't forget to update the instructions on the top-level readme to point people to the new build-project.sh script.

Reply:
done